cocos/core/renderer/scene/model.ts

```diff
@@ -88,166 +88,387 @@ const lightmapSamplerWithMipHash = new SamplerInfo(
+     * @zh 获取模型包围盒
+     */
     get modelBounds () {
         return this._modelBounds;
     }
 
+    /**
+     * @en return model ubo buffer
+     * @zh 获取模型 ubo 缓冲
+     */
     get localBuffer () {
         return this._localBuffer;
     }
 
+    /**
+     * @en return world bound buffer
+     * @zh 获取世界包围盒 ubo 缓冲
+     */
     get worldBoundBuffer () {
         return this._worldBoundBuffer;
     }
 
+    /**
+     * @en return update time stamp
+     * @zh 获取更新时间戳
+     */
     get updateStamp () {
         return this._updateStamp;
     }
 
+    /**
+     * @en is instancing enabled
+     * @zh 是否开启实例化
+     */
     get isInstancingEnabled () {
         return this._instMatWorldIdx >= 0;
     }
 
+    /**
+     * @en return shadow bias
+     * @zh 获取阴影偏移值
+     */
     get shadowBias () {
         return this._shadowBias;
     }
 
+    /**
+     * @en set shadow bias
+     * @zh 设置阴影偏移值
+     */
     set shadowBias (val) {
         this._shadowBias = val;
         if (JSB) {
             this._nativeObj!.setShadowBias(val);
         }
     }
 
+    /**
+     * @en return shadow normal bias
+     * @zh 获取阴影法线偏移值
+     */
     get shadowNormalBias () {
         return this._shadowNormalBias;
     }
 
+    /**
+     * @en set shadow normal bias
+     * @zh 设置阴影法线偏移值
+     */
     set shadowNormalBias (val) {
         this._shadowNormalBias = val;
         if (JSB) {
             this._nativeObj!.setShadowNormalBias(val);
         }
     }
 
+    /**
+     * @en return receive shadow or not
+     * @zh 返回是否接收阴影
+     */
     get receiveShadow () {
         return this._receiveShadow;
     }
 
+    /**
+     * @en set receive shadow or not
+     * @zh 设置是否接收阴影
+     */
     set receiveShadow (val) {
         this._setReceiveShadow(val);
         this.onMacroPatchesStateChanged();
     }
 
+    /**
+     * @en return cast shadow or not
+     * @zh 返回是否投射阴影
+     */
     get castShadow () {
         return this._castShadow;
     }
 
+    /**
+     * @en set cast shadow or not
+     * @zh 设置是否投射阴影
+     */
     set castShadow (val) {
         this._castShadow = val;
         if (JSB) {
             this._nativeObj!.setCastShadow(val);
         }
     }
 
+    /**
+     * @en return model's node
+     * @zh 返回模型所在的节点
+     */
     get node () : Node {
         return this._node;
     }
 
+    /**
+     * @en set model's node
+     * @zh 设置模型所在的节点
+     */
     set node (n: Node) {
         this._node = n;
         if (JSB) {
             this._nativeObj!.setNode(n.native);
         }
     }
 
+    /**
+     * @en return model's transform
+     * @zh 返回模型的变换
+     */
     get transform () : Node {
         return this._transform;
     }
 
+    /**
+     * @en set model's transform
+     * @zh 设置模型的变换
+     */
     set transform (n: Node) {
         this._transform = n;
         if (JSB) {
             this._nativeObj!.setTransform(n.native);
         }
     }
 
+    /**
+     * @en return model's visible tag
+     * @zh 返回模型的可见标志
```

Comment:
Model's visibility flags, it's different from [[Node.layer]], but it will also be compared with [[Camera.visibility]] during culling process.